### ISSUE TITLE
Replace French Comments with English ones

### ISSUE THREAD

**Issue by jounathaen** [blupi-games/planetblupi#31]
Some Sources (I have not checked all of them) seem to have comments in french language. As some people (like me) don't speak french, it would be great to have english comments in the source code.
Greetings!

**Comment by Skywalker13** [comment]
Yes, I already translated a lot of texts, not only the comments, but even enumerations, defines, etc... it's boring but it will be done step by step. And some comments are just wrong.
